docs/interpreter/mongodb.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: page
+title: "MongoDB Interpreter for Apache Zeppelin"
+description: "MongoDB is a general purpose, document-based, distributed database built for modern application developers and for the cloud era."
+group: interpreter
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+# MongoDB interpreter for Apache Zeppelin
+
+<div id="toc"></div>
+
+## Overview
+[MongoDB](https://www.mongodb.com/) is a general purpose, document-based, distributed database built for modern application developers and for the cloud era.
+This interpreter use mongo shell to execute [scripts](https://docs.mongodb.com/manual/tutorial/write-scripts-for-the-mongo-shell/)
```

Comment:
Miss dot at the end of this sentence. This is what I see in web 
![image](https://user-images.githubusercontent.com/164491/72808201-04b4b700-3c94-11ea-932e-42c0500523a7.png)